Packs/AccentureCTI/Integrations/ACTIVulnerabilityQuery/ACTIVulnerabilityQuery.py

```diff
@@ -0,0 +1,294 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+import requests
+import traceback
+from typing import List
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+''' CONSTANTS '''
+DATE_FORMAT = '%Y-%m-%d %H:%M:%S'
+ENDPOINTS = {
+    'vulnerability': '/rest/vulnerability'
+}
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    def __init__(self, input_url: str, api_key: str, verify_certificate: bool, proxy: bool, endpoint="/rest/vulnerability"):
+        base_url = urljoin(input_url, endpoint)
+        headers = {
+            "Content-Type": "application/json",
+            'auth-token': api_key
+        }
+        super(Client, self).__init__(base_url=base_url,
+                                     verify=verify_certificate,
+                                     headers=headers,
+                                     proxy=proxy)
+
+    def vulnerability_search(self, url_suffix: str, data: dict = {}) -> dict:
+        return self._http_request(method='GET', url_suffix=url_suffix, params=data)
+
+
+class CVE():
```

Comment:
please inherent from the default CVE class and add the extra fields there:
```
class CustomCVE(CVE):
``

Reply:
![image](https://user-images.githubusercontent.com/40510780/152524637-6c7741ce-cc48-4e1f-a407-02b22ca9d891.png)
@ilaner Don't know if this can work, but for now it has some error as its not able to identify the parent class possibly:
![image](https://user-images.githubusercontent.com/40510780/152524874-430d16ac-1495-433b-b50e-a1a7e102def7.png)